src/Symfony/Component/Messenger/Transport/AmqpExt/AmqpTransportFactory.php

```diff
@@ -34,7 +34,13 @@ public function __construct(EncoderInterface $encoder, DecoderInterface $decoder
 
     public function createTransport(string $dsn, array $options): TransportInterface
     {
-        return new AmqpTransport($this->encoder, $this->decoder, $dsn, $options, $this->debug);
+        return new AmqpTransport(
```

Comment:
According to the Code Standard shouldn't it be in one line?

Reply:
yes, it should